students/rmart300/lesson07/test_html_render.py

```diff
@@ -0,0 +1,109 @@
+from html_render import * 
+import unittest
+
+class test_element(unittest.TestCase):
+
+    
+
+    def test_html(self):
+        html = Html()
+        assert html.opening_tag == "<!DOCTYPE html>\n<html>\n"
+        assert html.closing_tag == "</html>\n"
+        assert html.attr_str == ""
+
+    def test_body(self):
+        body = Body()
+        tag = body.tag
+        cur_ind = body.cur_ind 
+        assert body.opening_tag == f"{cur_ind}<{tag}>\n"
+        assert body.closing_tag == f"{cur_ind}</{tag}>\n"
+        assert body.attr_str == ""
```

Comment:
it is preferred for "expected" thing to not rely on the class you're testing, for example below would be better:
```
assert body.opening_tag == "<sometag>\n"
```